### PR TITLE
refactor(swap): clean up Swap widget from isWrapOrUnwrap

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Context: https://github.com/cowprotocol/cowswap/pull/2774

Removed `wrap/unwrap` checks from Swap page code, because `wrap/unwrap` flow is consolidated in `TradeWidget`.

# To Test (Swap / Limit / Advanced)

1. Wrap/Unwrap flow should work the same in all trade widgets
2. ETH-flow (only for Swap)

### CODE REVIEW

File: src/modules/swap/containers/ConfirmSwapModalSetup/index.tsx [hunk 1]
```diff
@@ -70,14 +56,6 @@ export function ConfirmSwapModalSetup(props: ConfirmSwapModalSetupProps) {
...
         onDismiss={handleConfirmDismiss}
       />
-
-      <TransactionConfirmationModal
-        attemptingTxn={true}
-        isOpen={showTransactionConfirmationModal}
-        pendingText={pendingText}
-        onDismiss={onDismiss}
-        operationType={operationType}
-      />
```
Comment (alfetopito) [inline src/modules/swap/containers/ConfirmSwapModalSetup/index.tsx:80]: Ah, I see where this was being used!

**Review by alfetopito — APPROVED** [review]
Looks and works good 👍 

Only one behaviour change from prod which might be a problem, but not a big one.
It's no longer possible to insert the receive amount when wrapping/unwrapping.
Yes, the amounts are exactly the same, it has no actual difference from inputting it on sell or buy, but raising it just in case.

**Review by anxolin — APPROVED** [review]
Great to see so many deletions of lines of code!

Soft approve, cause i haven't tested it. I would suggest @elena-zh tests this, but not the previous one. Because we might need to do quite some tests

**Comment by shoom3301** [comment]
@elena-zh thanks! Fixed everything

**Comment by shoom3301** [comment]
>not allowed to edit TO field's value

I did it intentionally to keep the code simple. I don't see any concrete reasons for this functionality

**Comment by elena-zh** [comment]
Hey @shoom3301 , all is good, The last issue that I see that there is no balance check on Wrap/unwrap transaction. Could you please fix this as well?
no balance


 Wrap/unwrap button should be disabled when 'could not load balances' and 'insufficient balance'  errors are happening.

**Comment by elena-zh** [comment]
Hey @shoom3301 , the last nitpick: amount is reset to 0 when press on the change tokens arrow on the Swap page: 
image

**Comment by shoom3301** [comment]
@elena-zh thanks, fixed!

**Review by elena-zh — APPROVED** [review]
LGTM!

File: src/modules/tradeFormValidation/pure/TradeFormButtons/tradeButtonsMap.tsx [hunk 2]
```diff
@@ -57,14 +57,11 @@ const unsupportedTokenButton = (context: TradeFormButtonContext) => {
...
 export const tradeButtonsMap: Record<TradeFormValidation, ButtonErrorConfig | ButtonCallback> = {
-  [TradeFormValidation.WrapUnwrapAmountNotSet]: {
-    text: 'Enter an amount',
-  },
   [TradeFormValidation.WrapUnwrapFlow]: (context) => {
     const isNativeIn = !!context.derivedState.inputCurrency?.isNative
 
     return (
-      <TradeFormBlankButton onClick={context.wrapNativeFlow}>
+      <TradeFormBlankButton onClick={() => context.wrapNativeFlow()}>
```
Comment (alfetopito) [inline src/modules/tradeFormValidation/pure/TradeFormButtons/tradeButtonsMap.tsx:64]: Why do you need the wrapper fn?

**Pull request merged**